conda/gateways/logging.py

```diff
@@ -21,8 +21,13 @@
 from ..common.io import _FORMATTER, attach_stderr_handler
 
 log = getLogger(__name__)
-TRACE = 5  # TRACE LOG LEVEL
-VERBOSITY_LEVELS = (WARN, INFO, DEBUG, TRACE)
+VERBOSITY_LEVELS = {
+    0: WARN,  # standard output
+    1: WARN,  # -v, detailed output
+    2: INFO,  # -vv, info logging
+    3: DEBUG,  # -vvv, debug logging
+    4: (TRACE := 5),  # -vvvv, trace logging
+}
```

Comment:
I think this translation between `logging` and our CLI/logging setup will invite incompatibility when integrating plugins into conda. For better or worse, we need to use the logging levels the logging library provides, as it's the lowest common denominator when writing stdlib based packages.

In logging lore, `0` means `NOTSET`, meaning that everything should be logged. I think that's close to what we mean as “standard output”.

Reply:
this mapping is marked for deprecation, see `context.log_level` instead